src/flycapture2.pyx

```diff
@@ -385,6 +394,165 @@ cdef class Context:
         with nogil:
             r = fc2SetConfiguration(self.ctx, &config)
         raise_error(r)
+        
+        
+    ############################################################
+    # GigE support
+
+    def set_byteorder_littleendian(self):
+        '''Get byte order        
+        '''
+        cdef fc2Error r
+        cdef unsigned int regval = 0        
+        
+        with nogil:    
+            r = fc2WriteRegister(self.ctx, 0x1048, regval)
```

Comment:
It's a fast way of asking for little endian data. And this is necessary when working with 16 bits in gige, otherwise data is big endian and scrambled by default .

If this higher level function is removed, a lot of people will have this problem and will not know how to fix. If you REALLY insist we can remove that, and inform somewhere about the command that must be run :  fc2WriteRegister(ctx, 0x1048, 0).